Application/EdFi.Ods.Common/Infrastructure/Extensibility/EntityExtensionRegistrar.cs

```diff
@@ -93,6 +95,7 @@ private void RegisterAggregateExtensionEntity(Type edFiStandardEntityType, Entit
             bool isRequired)
         {
             _logger.Debug($"updating {nameof(edFiStandardEntityType)} with extension {extensionName}");
```

Comment:
Add single quotes around the parameters in the log message.

Reply:
done